tests/api/items/get_by_bbox.test.ts

```diff
@@ -0,0 +1,31 @@
+import 'should'
+import { map } from 'lodash-es'
+import { createItem } from '#fixtures/items'
+import { createUser, getRandomPosition } from '#fixtures/users'
+import { fixedEncodeURIComponent } from '#lib/utils/url'
+import { waitForIndexation } from '#tests/api/utils/search'
+import { publicReq } from '#tests/api/utils/utils'
+import type { LatLng } from '#types/common'
+
+const endpoint = '/api/items?action=by-bbox'
+
+const userPosition: LatLng = getRandomPosition()
+const [ lat, lng ] = userPosition
+const geolocatedUserPromise = createUser({ position: userPosition })
```

Comment:
Please avoid creating fixtures outside of test functions, as it means those are created even if the test are ignored/xitted, which adds noise to the tests logs. If sharing fixtures between tests is needed, such variable can be initialized from a `before` function, see [example](https://github.com/inventaire/inventaire/blob/377c5bf/tests/api/entities/serie_parts.test.ts#L11)

Reply:
Thanks for the example. Done in 32114a27e